certification/artifacts/artifacts.go

```diff
@@ -17,7 +18,7 @@ func WriteFile(filename, contents string) (string, error) {
 
 	err := os.WriteFile(fullFilePath, []byte(contents), 0o644)
 	if err != nil {
-		return fullFilePath, err
+		return fullFilePath, fmt.Errorf("could not write file to artifacts diretory: %v", err)
```

Comment:
Should this be `%w` for consistency? `%v and %w` are synonyms, but maybe we should pick one and stick with it?

Reply:
`%v` means `default` so I assume in the [docs](https://pkg.go.dev/fmt#Errorf) just mean `%w` is the same as `default`, at least that's how i read it.

Reply:
They aren't exactly synonyms. %w allows for unwrapping the error, whereas %v does not. Not sure what you mean by default.

Reply:
And %v was chosen precisely so that the error type doesn't leak out of the package. If %w is returned from a package export, it will end up being a part of the API, and we do not want to do that unless we are explicit about it.